builder/sizes_test.go

```diff
@@ -41,9 +41,9 @@ func TestBinarySize(t *testing.T) {
 	// This is a small number of very diverse targets that we want to test.
 	tests := []sizeTest{
 		// microcontrollers
-		{"hifive1b", "examples/echo", 4484, 280, 0, 2252},
-		{"microbit", "examples/serial", 2732, 388, 8, 2256},
-		{"wioterminal", "examples/pininterrupt", 6016, 1484, 116, 6816},
+		{"hifive1b", "examples/echo", 4504, 280, 0, 2252},
+		{"microbit", "examples/serial", 2736, 388, 8, 2256},
+		{"wioterminal", "examples/pininterrupt", 6020, 1484, 116, 6816},
```

Comment:
That's unfortunate :(
Not sure why this is happening, I'm guessing something with reflect and maybe interface comparisons?

Did you check which commit caused these changes? It would be interesting to know (and perhaps should be squashed together with that commit).

Reply:
This is almost certainly seeding-runtime-fastrand change, but I'll double check.

Reply:
Right, I see.